### ISSUE TITLE
governing structure

### ISSUE THREAD

**Issue by troettge** [troettge/RN-Norway#14]
I still go back and forth regarding our next steps. I talked to Hans Rocha IJzerman today about a grant that I want to write and we started chatting about his efforts on the French RN. He said that they go the more democratic route and try not to alienate people by initial exclusion.

So I am still for a paragraph displayed right on the landing page that clarifies (a) that the network is an unrealized idea that this website is supposed to help manifesting, (b) what the next steps are (e.g. identifying local initiatives, collecting interested parties across disciplines and locations, concrete next steps) and (c) that people that want to be involved to get in touch with us. 

I also realized that we really need to develop a good idea of what this network should look like in the near and far future. This will determine the governing structure tremendously. I think having a plan A for where this is going within the next 6 months would be really valuable to not only clarify the journey for us but also to offer a road map to interested parties. (sorry for the rambling, it's been a long day).

I added this point to the [agenda](https://github.com/troettge/RN-Norway/issues/13) for next meeting, but we can discuss this asynchronously here already.

**Comment by julienmayor** [comment]
Yep - and that would also affect multiple docs that we want to put online - e.g., the code of conduct (who acts on issues, etc).
